index.html

```diff
@@ -2031,6 +2051,62 @@ <h2>
           Visual representation of a [=Standard Gamepad=] layout.
         </figcaption>
       </figure>
+      <section data-dfn-for="ExtendedMapping" data-link-for="ExtendedMapping">
+        <h3>
+	  <dfn>ExtendedMapping</dfn>
+        </h3>
+        <p>
+          This section introduces an extended gamepad button mapping beyond the Standard Gamepad layout.
+          These additional buttons are commonly found on certain gamepad models.
+          The following table defines the extended buttons used by some gamepad models.
+          It's important to note that this list is not exhaustive, and user agents may utilize different
+          or additional buttons for these or other gamepad models.
+          Consequently, the number of buttons on the {{Gamepad}} is not limited to the standard mapping of 17 buttons.
+        </p>
+       <p>
+          To accommodate extended gamepads, we have incorporated an "extended" mapping enumeration
+          into the {{GamepadMappingType}}. Additionally, we have defined an enumeration for the various
+          button types termed {{GamepadButtonType}}, and have expanded the {{GamepadButton}}
+          interface to encompass this new button type enumeration.
+       </p>
+       <section>
+          <h3>
+            <dfn>GamepadButtonType</dfn> Enum
+          </h3>
+          <p>
+            This enum defines the set of possible button types.
+          </p>
+          <pre class="idl">
+            enum GamepadButtonType {
+              "standard",
+              "non-canonical",
+               "trackpad", 
+               // ... More enum types to be defined by the separate W3C wiki page.
```

Comment:
We can do this, but we need to change GamepadButtonType to an DOMString. Otherwise, the enum will always be out of date. Alternatively, we don't use the Wiki and just use the spec... it might be better to just use the spec, so we can have more oversight over what is defined here (and get better interop).